homework_07.py

```diff
@@ -0,0 +1,182 @@
+""".
+
+Задача: надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити або доповнити.
+
+"""
+
+
+def multiplication_table(number):
+    """task1."""
+    # Initialize the appropriate variable
+    multiplier = 1
+    # Complete the while loop condition.
+    while True:
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if result > 25:
+            # Enter the action to take if the result is greater than 25
+            break
+        print(str(number) + 'x' + str(multiplier) + '=' + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+
+
+multiplication_table(3)
+# Should print:
+# 3x1=3
+# 3x2=6
+# 3x3=9
+# 3x4=12
+# 3x5=15
+
+iii = '-'
+
+print(iii * 20)
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+
+
+def sumoftwo(a, b):
+    """Find sum of 2 number."""
+    return a + b
+
+
+# task 3
+
+
+"""Написати функцію, яка розрахує середнє арифметичне списку чисел.
+"""
+
+numlists = [1, 2, 3, 4, 5, 6]
+
+
+def avar_num(numlists):
+    """Task 3."""
+    return sum(numlists) / len(numlists)
+
+
+# task 4
+
+
+"""  Написати функцію, яка приймає рядок та повертає його у зворотному порядку.
+"""
+
+
+def rev_str(a):
+    """Reverse of the string."""
+    return a[::-1]
+
+
+# task 5
+
+
+"""Написати функцію, яка приймає список слів та повертає найдовше
+слово у списку.
+"""
+b = ['hello', 'world', 'hellooooo']
+
+
+def long_word(b):
+    """Longest word from list."""
+    return max(b, key=len)
+
+
+# task 6
+
+
+"""  Написати функцію, яка приймає два рядки та повертає
+ індекс першого входження другого рядка
+у перший рядок, якщо другий рядок є підрядком першого
+ рядка, та -1, якщо другий рядок
+не є підрядком першого рядка."""
+
+
+def find_substring(str1, str2):
+    """task6."""
+    return -1
```

Comment:
Incorrect. If we sub2 within sub1 - we should return index of this occurrence. Now you jsut hardcoded "-1"